Policies/github-corporate-terms-of-service.md

```diff
@@ -144,12 +144,14 @@ Because you retain ownership of and responsibility for Your Content, we need you
 #### 4. License Grant to Us
 We need the legal right to do things like host Your Content, publish it, and share it. You grant us and our legal successors the right to store, parse, and display Your Content, and make incidental copies as necessary to render the Website and provide the Service. This includes the right to do things like copy it to our database and make backups; show it to you and other users; parse it into a search index or otherwise analyze it on our servers; share it with other users; and perform it, in case Your Content is something like music or video.
 
-This license does not grant GitHub the right to sell Your Content or otherwise distribute or use it outside of our provision of the Service.
+This license does not grant GitHub any additional rights not expressly granted above unless granted by a [project license](/articles/adding-a-license-to-a-repository/#including-an-open-source-license-in-your-repository) or other terms and conditions applicable to your content.
```

Comment:
The old line is [also present in `github-terms-of-service.md`][1].  I recommend keeping those two places in sync.  This applies to the second hunk you alter as well.

[1]: https://github.com/github/site-policy/blame/1263bf0164cfe60d0900e70ba9126855c7c23c6c/Policies/github-terms-of-service.md#L142

Reply:
I didn't expect this would necessarily be merged directly, rather taken as a suggestion to Github for future TOS updates. It would ultimately be up to Github where the 2 TOS need to align.

Reply:
Ok.  Thank you